package/boot/arm-trusted-firmware-mvebu/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2019 OpenWrt
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arm-trusted-firmware-mvebu
+PKG_RELEASE:=2.1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_RELEASE).tar.xz
+PKG_SOURCE_URL:=https://git.trustedfirmware.org/TF-A/trusted-firmware-a.git
+PKG_SOURCE_DATE:=2019-10-21
+PKG_SOURCE_VERSION:=937f669872b5516c9a0fd3d60f9618ad54a72b5c
+PKG_MIRROR_HASH:=ca7e6b148f16064c99303b7e0f9197ecf20b61793b0a947fc454852629dabda1
+
+PKG_MAINTAINER:=Vladimir Vid <vladimir.vid@sartura.hr>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/arm-trusted-firmware-mvebu
+    SECTION:=boot
+    CATEGORY:=Boot Loaders
+    TITLE:=ARM Trusted Firmware for mvebu devices
+    DEPENDS:=@TARGET_mvebu
```

Comment:
you should probably restrict this to the 64 bit subtargets, I assume that this will not build on the 32 bit marvel subtargets

Reply:
@hauke that makes sense, changed target to mvebu64